ReachCare/core/migrations/0008_remove_userquestionnaire_next_response.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.0.5 on 2020-07-21 03:18
+
+from django.db import migrations
```

Comment:
Nit: Instead of adding the field and then dropping, can you delete both migrations and create a new one with just the one field?

Reply:
np. can do.